docs/v2-system-addon/snippets.md

```diff
@@ -76,3 +76,16 @@ For example:
 `appData.defaultBrowser`: (bool) Is Firefox the user's default browser?
 If we could not determine the default browser, this value is `null`. Note that
 currently this value is only checked once when the browser is initialized.
+
+## Events
+
+ActivityStream dispatches custom DOM events on the window to allow snippets
+providers to respond to ActivityStream snippets events.
+
+### `SnippetsEnabled`
```

Comment:
Maybe do `Snippets:Enabled` and `Snippets:Disabled` to follow the style of other events in the tree